centreon/tests/e2e/features/Cloud-notifications/02-notification-duplicate.feature

```diff
@@ -0,0 +1,28 @@
+@ignore
+@REQ_MON-24180
+Feature: Duplicating a notification rule
+  As a Centreon user with access to the notification rules page
+  I need to duplicate a notification rule
+  So that I can save time and configure a new one without starting from scratch
+
+  Background:
+    Given a user with access to the notification rules page
+    And the user is on the notification rules page
+    And a notification rule is already created
+
+  Scenario: Duplicating a notification rule
+    When the user selects the duplication action on a notification rule
+    And the user enters a new name
```

Comment:
isn t there a step about a specific menu or does the selection immediately triggers the action of duplication ?

Reply:
no, only action